tests/test_squash_command.rs

```diff
@@ -312,6 +312,46 @@ fn test_squash_description() {
     "###);
 }
 
+#[test]
+fn test_squash_empty() {
+    let mut test_env = TestEnvironment::default();
+    test_env.jj_cmd_success(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+    
+    test_env.jj_cmd_success(&repo_path, &["describe", "-m", "parent"]);
+    test_env.jj_cmd_success(&repo_path, &["new"]);
```

Comment:
Those two commands can be replaced by `jj commit -m parent`.

Reply:
Done, thank you!